homework-3/src/views/DoctorProfile.js

```diff
@@ -0,0 +1,242 @@
+import BoxDoctor from '../components/common/Boxes/StylBoxDoctor';
+import BoxFlexColumn from '../components/common/Boxes/StylBoxFlexColumn';
+import BoxPatientContent from '../components/common/Boxes/StylBoxPatientContent';
+import BoxBtnPatients from '../components/common/Boxes/StylBoxBtnPatients';
+import BoxPatientsList from '../components/common/Boxes/StylBoxPatientsList';
+import TitleName from '../components/common/Titles/StylTitleName';
+import TitlePatients from '../components/common/Titles/StylTitlePatients';
+import SubTitlePostion from '../components/common/Titles/StylSubTitlePosition';
+import AvatarActive from '../components/AvatarActive';
+import BtnPatients from './../components/BtnsComponents/BtnPatients';
+import CardPatient from '../components/CardPatientProfileDoctor/CardPatient';
+import Loader from './../components/Loader/Loader';
+
+import avatar1 from '../img/avatars/avatar-doctor.png';
+import avatar2 from '../img/avatars/patients-1.png';
+import avatar3 from '../img/avatars/patients-2.png';
+import avatar4 from '../img/avatars/patients-3.png';
+import avatar5 from '../img/avatars/patients-4.png';
+import avatar6 from '../img/avatars/patients-5.png';
+import avatar7 from '../img/avatars/patients-6.png';
+import avatar8 from '../img/avatars/patients-7.png';
+import avatar9 from '../img/avatars/patients-8.png';
+import avatar10 from '../img/avatars/patients-9.png';
+import avatar11 from '../img/avatars/patients-10.png';
+import avatar12 from '../img/avatars/patients-11.png';
+import avatar13 from '../img/avatars/patients-12.png';
+import logo from '../img/logo.png';
+
+const DoctorProfile = ({ title }) => {
+  React.useEffect(() => {
+    document.title = title || '';
+  }, [title]);
+
+  const [loading, setLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      await new Promise((r) => setTimeout(r, 2000));
+
+      setLoading((loading) => !loading);
+    };
+
+    loadData();
+  }, []);
+
+  if (loading) {
+    return <Loader />;
+  } else {
+    return (
+      <BoxPatients>
+        <BoxRow>
+          <img src={logo} alt='logo' />
+          <BoxDoctor>
+            <BoxFlexColumn>
+              <TitleName>Alla Robins</TitleName>
+              <SubTitlePostion>Doctor</SubTitlePostion>
+            </BoxFlexColumn>
+            <AvatarActive src={avatar1} alt='avatar-user' />
+          </BoxDoctor>
+        </BoxRow>
+        <BoxPatientContent>
+          <BoxBtnPatients>
+            <BtnPatients active={true} text='Patients' />
+            <BtnPatients active={false} text='Resolutions  ' />
+          </BoxBtnPatients>
+          <BoxRowPadding>
+            <TitlePatients>My patients</TitlePatients>
+            <BoxDoctor>
+              <BoxSearch />
+              <BoxFilter text='Sort by:' filter='Date' />
+            </BoxDoctor>
+          </BoxRowPadding>
+          <BoxPatientsList>
+            <CardPatient
```

Comment:
a lot of useless code here
create one component template and receive prop data where u can storage all info about the patient and map it 
ur code become clearer and readable

Reply:
Message above